src/path/Curve.js

```diff
@@ -2289,6 +2247,57 @@ new function() { // Scope for bezier intersection using fat-line clipping
         return pairs;
     }
 
+    /**
+     * Internal method to calculates the curve-time parameters where the curve
+     * is tangential to provided tangent.
+     * Tangents at the start or end are included.
+     *
+     * @param {Number[]} v curve values
+     * @param {Point} point the tangent to which the curve must be tangential
+     * @return {Number[]} at most two curve-time parameters, where the curve is
+     * tangential to the given tangent
+     */
+    function getTimesWithTangent(v, point)
+    {
```

Comment:
@sasensi this should be on the same line as `getTimesWithTangent(v, point)` :)

Reply:
Oops, sorry :)